spec/helper/revenuecat_internal_helper_spec.rb

```diff
@@ -443,6 +466,12 @@ def setup_stubs
       expect(File.read(tmp_test_file_path)).to eq('Testing changing text=4.1.3')
     end
 
+    it 'changes multiple occurences of old string with new string' do
```

Comment:
Note that this is new. This didn't really happened with the `sed` command where only the first occurence of a string got replaced. I think this is good but lmk if you think otherwise.

Reply:
yeah, probably safer to update all instances